src/foam/nanos/auth/ProxyUserLocatorService.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * @license
+ * Copyright 2021 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.nanos.auth',
+  name: 'ProxyUserLocatorService',
+
+  implements: [
+    'foam.nanos.auth.UserLocatorService'
+  ],
+
+  properties: [
+    {
+      class: 'String',
+      name: 'serviceName',
+      expression: function(delegate$serviceName) {
+        return delegate$serviceName
+      },
+      setter: function(n) {
+        this.delegate.serviceName = n;
+      },
+    },
```

Comment:
where is this used?

Reply:
No needed. We don't need a client for this service. I removed this. Thank you!